tests/test_utils.py

```diff
@@ -155,3 +156,11 @@ def test_parse_env_tags(self):
             mock.call("Malformed tag in tag pair '%s' from tag string '%s'.", "key2", "key,key2,key3"),
             mock.call("Malformed tag in tag pair '%s' from tag string '%s'.", "key3", "key,key2,key3"),
         ])
+
+    def test_random(self):
```

Comment:
Just write a pytest style test in a `tests/test_rand.py` or the like. We'll never find it here otherwise if we ever need to add more tests.